openlcs/packages/serializers.py

```diff
@@ -69,16 +70,28 @@ class Meta:
                   "copyright_detections"]
 
     def get_license_detections(self, obj):
-        license_keys = obj.get_license_detections().values_list(
-            'license_key', flat=True
-        )
-        return license_keys.distinct()
+        license_data = obj.get_license_detections().values_list(
+            'license_key', flat=True)
+        ld_url_prefix = 'http://{}/rest/v1/licensedetections/'.format(
+                        settings.HOSTNAME)
+        license_detections = dict()
+        for lk in license_data.distinct():
+            detail_url = '{}?license_key={}'.format(
+                        ld_url_prefix, urlquote(lk))
+            license_detections[lk] = detail_url
```

Comment:
Result from /rest/v1/licensedetections/?license_key=gpl-2.0 would be something like:
```
    "results": [
        {
            "id": 86761,
            "license_key": "gpl-2.0",
            ...
            "file_scan": 27047
        },
        ...
```
The end user needs to know the file path instead of a` file_scan` id, and there is no endpoint for the `file_scan`.

Reply:
Per discussion with @buweixin , the file scan detailed data(the file's id, swhid, and detector) would be added instead. Note: file_path could be added, but this data can only observed when a source query param is provided.